tests/test_endpoints/test_jobs.py

```diff
@@ -0,0 +1,114 @@
+import os
+import json
+from uuid import UUID, uuid4
+from app.db import get_test_db
+from tests.conftest import client_login
+from tests.users import regular_member, admin_member
+
+from tests.utils.authentication import admin_required, authentication_required
+
+db = get_test_db()
+
+new_job = {
+    "company": "TD",
+    "title": "Vil du være TD's nye nettside ansvarlig?",
+    "type": "Charity",
+    "img": "",
+    "tags": ["TypeScript", "React", "Docker", "MongoDB"],
+    "description_preview": "Vi trenger noen til å ta over tds nye nettsider!",
+    "description": "Vi trenger noen til å ta over TDs nettsider og videreutvikle produktet.",
+    "start_date": "2023-01-01T00:00:00.000Z",
+    "published_date": "2023-01-01T00:00:00.000Z",
+    "location": "Tromsø",
+    "link": "td-uit.no",
+    "due_date": "2023-01-01T00:00:00.000Z",
+}
+
+
+@admin_required("/api/jobs/", "post")
+def test_create_job(client):
+    # Test admin auth
+    access_token = client_login(
+        client, regular_member["email"], regular_member["password"])
+    headers = {"Authorization": f"Bearer {access_token}"}
+    response = client.post(
+        "/api/jobs/", json=new_job, headers=headers)
+    assert response.status_code == 403
```

Comment:
if thees just test admin credential, the admin required decorator does exactly that